File: Difficulty: Easy/Frequency of Elements/frequency-of-elements.cpp
class Solution {
  public:
    vector<vector<int>> countFreq(vector<int>& arr) {
        // code here
        int n=arr.size();
        unordered_map<int,int>freq;
        for(int ele:arr){
            freq[ele]++;
        }
        
        vector<vector<int>>ans;
        for(auto ele:freq){
            ans.push_back({ele.first,ele.second});
        }
        return ans;
    }
};

File: Difficulty: Easy/Frequency of Elements/frequency-of-elements.java
class Solution {
    public ArrayList<ArrayList<Integer>> countFreq(int[] arr) {
        // code here
        HashMap<Integer,Integer>mp=new HashMap<>();
        
        for(int el:arr){
            mp.put(el,mp.getOrDefault(el,0)+1);
        }
        
        ArrayList<ArrayList<Integer>>list=new ArrayList<>();
        
        for(Map.Entry<Integer,Integer>entry : mp.entrySet()){
            ArrayList<Integer>pair=new ArrayList<>();
            pair.add(entry.getKey());
            pair.add(entry.getValue());
            list.add(pair);
        }
        return list;
    }
}